cli/src/build.ts

```diff
@@ -160,6 +165,25 @@ export class BuildCommand extends Command {
         CARGO_PROFILE_RELEASE_LTO: false,
       })
     }
+
+    if (this.useZig && triple.platform == 'linux') {
+      const paths = envPaths('napi-rs')
+      const cpuArch = getCpuArch(triple.arch)
+      const linkerWrapper = join(paths.cache, `zig-cc-${triple.abi}.sh`)
+      mkdirSync(paths.cache, { recursive: true })
+      writeFileSync(
+        linkerWrapper,
+        `#!/bin/bash\nzig cc \${@/-lgcc_s/-lunwind} -target ${cpuArch}-linux-${triple.abi}\n`,
```

Comment:
Obviously this don't work on Windows yet.

Reply:
Windows support is a little trickier, see https://github.com/PyO3/maturin/blob/36e525bba838eb10af324ec8805bf715a2b37139/src/zig.rs#L51-L57